cowprotocol/cowswap
# feat(twap-orders): No. of parts field (2)

# Summary

This PR adds a `No. of parts` input field
The code structure and functionality will be updated in the future PR's
Doing this as a separate PR for a simpler review.
The second repeated Field will be slippage in the following PR.

We can skip QA review for now on this.

# To test
1. Open dev tools
2. In console paste `localStorage.setItem('advanced-orders', '1')` and press enter then reload page
3. You should now see Advanced tab
